src/main/java/com/yahoo/bullet/querying/PostAggregationOperations.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright 2018, Oath Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+package com.yahoo.bullet.querying;
+
+import com.yahoo.bullet.parsing.Computation;
+import com.yahoo.bullet.parsing.OrderBy;
+import com.yahoo.bullet.parsing.PostAggregation;
+import com.yahoo.bullet.postaggregations.ComputationStrategy;
+import com.yahoo.bullet.postaggregations.OrderByStrategy;
+import com.yahoo.bullet.postaggregations.PostStrategy;
+
+public class PostAggregationOperations {
```

Comment:
Do we need this whole class for just one Static method?  Can we just put this static method in PostStrategy?

I know this maintains symmetry with the AggregationOperations class, which just has a single static "findStrategy" method - but why don't we put that method in the Strategy interface?  Why did we have a whole AggregationOperations class for a single static method that could just be in Strategy?

Reply:
It's consistent with AggregationOperations and WindowingOperations.  I guess we just wanted to keep all query operations are in the same package at the begin. If we want to change, we should change all of them in a separate PR.